src/bindings/js/node/tests/basic.test.js

```diff
@@ -14,6 +15,12 @@ const compiledModel = core.compileModelSync(model, 'CPU');
 const modelLike = [[model],
   [compiledModel]];
 
+it('Core.getAvailableDevices()', () => {
+    if (platform !== 'linux') return;
```

Comment:
Why is it linux feature only?

Reply:
Test was prepared for a currently supported platform. 
Will be removed if tests shouldn't include limitations of the compilation side.